demo/README.md

```diff
@@ -60,9 +60,7 @@ This is a list of short codes introducing different functionalities of xgboost p
 Most of examples in this section are based on CLI or python version.
 However, the parameter settings can be applied to all versions
 
-- [Binary classification](binary_classification)
 - [Multiclass classification](multiclass_classification)
-- [Regression](regression)
```

Comment:
The page was moved to `CLI/regression`. Can you update this link to `CLI/regression`?